mne/time_frequency/tfr.py

```diff
@@ -2452,7 +2456,7 @@ def _preproc_tfr(data, times, freqs, tmin, tmax, fmin, fmax, mode,
     data = data[:, ifmin:ifmax, itmin:itmax]
 
     if dB:
-        data = 10 * np.log10((data * data.conj()).real)
+        data = 10 * np.log10(data)
```

Comment:
If you have complex data this step is unchanged from before (the thing being `log10`ed here is a power, not amplitude). But if you have real data it's now wrong, as the thing being `log10`ed here is an amplitude not a power.

The easiest way to resolve this is with two changes, one here and one above, by keeping everything as amplitude and then using that knowledge here
```suggestion
        data = 20 * np.log10(data)
```

Reply:
Good catch!!